static/js/components/PersonalTab.js

```diff
@@ -45,30 +46,34 @@ class PersonalTab extends ProfileTab {
   }
 
   render() {
-    return <div>
-      {this.boundTextField(["first_name"], "Given name")}<br />
-      {this.boundTextField(["last_name"], "Family name")}<br />
-      {this.boundTextField(["preferred_name"], "Preferred name")}<br />
-      {this.boundSelectField(['gender'], 'Gender', this.genderOptions)}<br />
-      {this.boundSelectField(
+    return <Grid>
```

Comment:
you may want to add `className="profile-tab-grid"` here, so the spacing and width is in line with the other tabs.